docs/source/checks/simple.rst

```diff
@@ -0,0 +1,175 @@
+.. _simple:
+
+Simple Check
+====================
+
+Description
+-----------
+
+The SimpleCheck gives you basic information about your datasets and detects the most obvious of datashifts.
+
+Some shifts result in the change of basic metrics, that are commonly used for the analysis and exploration of datasets.
+Those metrics and statistics are calculated on both datasets and compared afterwards. HIER NOCH AMAZON DEEQU ERWÄHNEN,
+WIRD NACH UMBENENNUNG DES CHECKS GEMACHT
+
+The basic check should be used in most analyses because its metrics are scientifically grounded and commmonly used.
+It works directly on categorical and numerical data and, with additional Precalculations, also on text data.
+
+
+Example
+-------
+
+::
+
+    from shift_detector.detector import Detector
+    from shift_detector.checks.simple_check import SimpleCheck
+
+    data_set_1 = 'examples/pokedex1.csv'
+    data_set_2 = 'examples/pokedex2.csv'
```

Comment:
Until now, the documentation uses `shoes_first.csv` and `shoes_second.csv`... I think it is a good idea to have an example, the user can reproduce with data provided data, but the documentation should be consistent. So please change this or create an issue for the other checks

Reply:
created an issue #126